### PR TITLE
Changed docs.microsoft.com to learn.microsoft.com

### PR DESCRIPTION
**Pull request by mikehoffms**
Please inspect removal of /en-us in code listings.

### CODE REVIEW

File: src/cdpTargetsProvider.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export class CDPTargetsProvider implements vscode.TreeDataProvider<CDPTarget> {
         }
         const faviconRegex = /((?:\/\/|\.)([^\.]*)\.[^\.^\/]+\/).*/;
 
-        // Example regex match: https://docs.microsoft.com/en-us/microsoft-edge/
+        // Example regex match: https://learn.microsoft.com/en-us/microsoft-edge/
```
Comment (mikehoffms) [inline src/cdpTargetsProvider.ts:110]: @codepo8 - retained "/en-us" bc also on next line

Comment (codepo8) [reply src/cdpTargetsProvider.ts:110]: You can remove the en-us without a problem, this makes no difference to the regex here.

**Review by codepo8 — APPROVED** [review]
Works for me.

**Review by codepo8 — APPROVED** [review]

**Comment by captainbrosset** [comment]
@codepo8 should we merge this now?

**Comment by codepo8** [comment]
sure

From: Patrick Brosset ***@***.***>
Date: Tuesday, 25 October 2022 at 10:18
To: microsoft/vscode-edge-devtools ***@***.***>
Cc: Chris Heilmann ***@***.***>, Mention ***@***.***>
Subject: Re: [microsoft/vscode-edge-devtools] Changed docs.microsoft.com to learn.microsoft.com (PR #1230)

@codepo8<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcodepo8&data=05%7C01%7Cchristian.heilmann%40microsoft.com%7Ccbce67c4183d412f066308dab6618878%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638022827308512524%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=rr7edLjI7QAfUdfB0sdR4EATu1%2FSfXlBYVIzHD6CR0w%3D&reserved=0> should we merge this now?

**Comment by codepo8** [comment]
yes, merge it.

**Pull request merged**